src/module/rules/rule-element/item-alteration/handlers.ts

```diff
@@ -966,29 +977,47 @@ const ITEM_ALTERATION_HANDLERS = {
                 required: true,
                 choices: ["add", "remove", "subtract"],
             }),
-            value: new fields.StringField<ItemTrait, ItemTrait, true, false, false>({
-                required: true,
-                nullable: false,
-                initial: undefined,
-            }),
+            value: new DataUnionField<TraitsValueField, true, false>(
+                [
+                    new fields.SchemaField({
+                        trait: new fields.StringField({ required: true, nullable: false }),
+                        annotation: new ResolvableValueField({ required: true, nullable: false }),
+                    }) satisfies TraitsValueConfigField,
+                    new StrictStringField<ItemTrait, ItemTrait, true, false, false>({
+                        required: true,
+                        nullable: false,
+                        initial: undefined,
+                    }),
```

Comment:
This is just a `StrictStringField` of `string`, right?

Reply:
Ya. It was a StringField for string that claimed it was for Item trait before. I think the original intent was that the validateForItem will validate that it's an ItemTrait

Reply:
If we remove the ItemTrait that'll let me upgrade this to remove "volley" without having to specify for what, but we'd lose validation for subtract and we'd need to double validate add so not sure about that

Reply:
I just realized that I'm ambiguous. Nah, it is a StrictStringField of ItemTrait, the validation is just deferred. This isn't a change from before other than adding Strict cause it's a data union field.

Reply:
Isn't that true for every single stringfield ever?